estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,51 @@
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offer"
+
+    price = fields.Float()
+    status = fields.Selection(
+        selection=[("accepted", "Accepted"), ("refused", "Refused")],
+        copy=False,
+    )
+    partner_id = fields.Many2one("res.partner", string="Buyer",  required=True)
```

Comment:
linter cries about this space 😅 

```suggestion
    partner_id = fields.Many2one("res.partner", string="Buyer", required=True)
```